spec/rototiller/task/params/command_spec.rb

```diff
@@ -63,7 +63,7 @@ module Task
         end
         context 'with a block' do
           it 'runs the block' do
-            expect{ command.run { |result| puts "my exit_code: '#{result.exit_code}'" } }.to output("#{@arg_name}\nmy exit_code: '0'\n").to_stdout
+            expect{ command.run { |result| puts "my exit_code: '#{result.exit_code}'" } }.to output(/#{@arg_name}\nmy exit_code: '0'\n/).to_stdout
```

Comment:
It used to just print out a message because the command is an echo command.  Now it prints the entire command, and then the echo'd message.  I thought about using .end_with? but that doesn't work with the output method.  I guess I could change the regex to ensure it's at the end.